### PR TITLE
PL: Some tweaks to workshop emails & subjects

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: dashboard/app/mailers/pd/workshop_mailer.rb [hunk 1]
```diff
@@ -206,7 +206,7 @@ def exit_survey(enrollment)
 
     mail content_type: content_type,
       from: from_survey,
-      subject: 'How was your Code.org workshop?',
+      subject: "Help us improve Code.org #{workshop.course} workshops!",
```
Comment (islemaster) [inline dashboard/app/mailers/pd/workshop_mailer.rb]: Should that be?
```suggestion
      subject: "Help us improve Code.org #{@workshop.course} workshops!",
```

Comment (breville) [reply dashboard/app/mailers/pd/workshop_mailer.rb]: Oops yes, thanks!

**Review by islemaster — APPROVED** [review]
:+1: :shipit:

File: dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml [hunk 2]
```diff
@@ -61,3 +61,9 @@
   P.S. Connect with other computer science teachers on Twitter
   = link_to '@TeachCode', 'https://twitter.com/teachcode', target: "_blank"
   for inspiration and great tips on bringing computer science to every school. Speaking of social media, if you don't want your photo taken or posted at the workshop, make sure the facilitator knows before the workshop begins.
+
+  P.S. Connect with other computer science teachers on Twitter
```
Comment (molly-moen) [inline dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml]: should the previous paragraph be deleted?

Comment (breville) [reply dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml]: Great catch!

**Review by molly-moen — APPROVED** [review]

**Pull request merged**